package.json

```diff
@@ -26,10 +26,10 @@
   "licenseFilename": "LICENSE",
   "readmeFilename": "README.md",
   "peerDependencies": {
-    "react-native": "^0.59.0"
+    "react-native": "0.61.0"
```

Comment:
Let's use the current latest `0.61.2` - there are some important fixes (along with issues unfortunately)

Reply:
👍